jung-algorithms/src/test/java/edu/uci/ics/jung/algorithms/shortestpath/TestShortestPath.java

```diff
@@ -479,35 +478,35 @@ public String get() {
 
   @Override
   protected void setUp() {
-    edgeWeights = new HashMap<Integer, Number>();
+    edgeWeights = new HashMap<>();
     nev = edgeWeights::get;
     dg = NetworkBuilder.directed().allowsParallelEdges(true).allowsSelfLoops(true).build();
-    for (int i = 0; i < dg_distances.length; i++) {
+    for (double[] dg_distance : dg_distances) {
```

Comment:
I don't think that in this case (or the one below on line 492) that it's actually a good idea to make this a foreach loop.

This is not actually doing _anything_ with the `dg_distances` array; it's just saying "do [X] as many times as there are elements in the dg_distances array".  That is: we don't need a `dg_distance` variable, and so creating it seems a bit odd.

Reply:
Good point! I hadn't noticed that when I created this change. (It was suggested amongst many other potential changes in my IntelliJ when I ran a manual _"Inspect Code..."_, so it slipped my notice that foreach could've been a bad idea here.)

Reverting. :)